conf/application.conf

```diff
@@ -43,7 +43,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
 
 # Json error handler
-play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.backend.http.JsonErrorHandler"
```

Comment:
I understand what you've done here but we should be careful about swapping out the handler that HMRC uses -- it might be okay, but I'd rather see what it is doing and whether we need to be aware of any of this before we just swap it out.

Reply:
Ah, no, you've extended the class. Okay, I missed that, that should be okay in this scenario.